data_steward/tools/add_hpo.py

```diff
@@ -2,7 +2,7 @@
 Add a new HPO site to config file and BigQuery lookup tables and updates the `pipeline_table_sandbox.site_masking`
     sandbox table created in `generate_site_mappings_and_ext_tables` with any missing hpo_sites in
```

Comment:
Can you update the file description to not mention `generate_site_mappings_and_ext_tables`?